cypress/integration/pomAllGalleries.spec.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+
+import { allGalleries } from "../page_objects/allGalleries";
+import { myGalleries } from "../page_objects/myGalleries";
+import { authLogin } from "../page_objects/authLogin";
+import { createGallery } from "../page_objects/createGallery";
+import { header } from "../page_objects/header";
+
+
+describe('POM all galleries', () => {
+
```

Comment:
Naziv describe bloka bi trebao da nas upucuje na to sta se testira u konkretnom .spec-u, `POM all galleries` nam ne govori mnogo, `test all gallery page` na primer bi bilo prikladnije.